src/entity-manager/MongoEntityManager.ts

```diff
@@ -87,25 +100,51 @@ export class MongoEntityManager extends EntityManager {
     /**
      * Finds entities that match given find options.
      */
-    async find<Entity>(
-        entityClassOrName: EntityTarget<Entity>,
-        options?: MongoFindManyOptions<Entity>,
-    ): Promise<Entity[]> {
-        return this.executeFind(entityClassOrName, options)
-    }
-
     /**
-     * Finds entities that match given conditions.
+     * Finds entities that match given find options or conditions.
      */
-    async findBy<Entity>(
+    async find<Entity>(
         entityClassOrName: EntityTarget<Entity>,
-        where: any,
+        optionsOrConditions?: FindManyOptions<Entity> | Partial<Entity>,
     ): Promise<Entity[]> {
```

Comment:
to match `mongodb`'s types, should the return type here be `Promise<WithId<Entity>[]>`? same goes for all the `MongoRepository.find*` and `MongoManager.find*` methods + `MongoRepository.createEntityCursor`

https://github.com/mongodb/node-mongodb-native/blob/d55022bfadfbddb16bf54bd9a801d901e0af99a0/src/collection.ts#L761